modules/core/client/app/config.js

```diff
@@ -1,23 +1,29 @@
-'use strict';
+(function (window) {
```

Comment:
Just a thought since you are now sending window, may as well make it a true IIFE and add angular.